.github/labeler.yml

```
@@ -0,0 +1,29 @@
+area/register:
+  - 'cmd/register/*'
+  - 'pkg/config/*'
+  - 'pkg/tpm/*'
+
+area/tests:
+  - 'tests/*'
+  - 'tests/**'
+  - '*/**/*_test.go'
+
+area/operator:
+  - 'pkg/**/*'
+  - 'cmd/operator/*'
```

🤔 may it be convenient to add also the register and, why not, the support cmds to the `operator` area?
```suggestion
  - 'cmd/**/*'
```